code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -226,6 +226,11 @@ datum/reagent/medicine/adminordrazine/on_mob_life(var/mob/living/carbon/M as mob
 	..()
 	return
 
+datum/reagent/medicine/adminordrazine/nanites
```

Comment:
It unbalances the other admin events by giving this one a automatic powerful healing tool which basically none of the others have at all........
The only other one that has actual healing are xenos and they are randomly sent to the station by events anyhow.....